common/siklu/siklu_spbs_command.c

```diff
@@ -13,43 +13,16 @@ typedef enum
 	RESET_MAX_OPTIONS
 } RESET_OPTION_TYPE;
 
-const char CTU_SPBS_NAME[] = "ap_gpio071";
+const int CTU_SPBS_OFFSET = 71;
 
 static RESET_OPTION_TYPE siklu_get_reset_status(void)
 {
 	RESET_OPTION_TYPE ret = RESET_MAX_OPTIONS;
-	unsigned gpio_no;
+	int gpio_no = CTU_SPBS_OFFSET;
 	const char * failure = NULL;
 	int reset_status;
-	const char * spbs_name = NULL;
 
-	if(of_machine_is_compatible("siklu,n366"))
-	{
-		spbs_name = CTU_SPBS_NAME;
-	}
-	else
-	{
-		failure = "non-suported board";
-		goto error;
-	}
-
-	if(gpio_lookup_name(spbs_name, NULL, NULL, &gpio_no))
-	{
-		failure = "gpio_lookup_name";
-		goto error;
-	}
-
-	if(gpio_request(gpio_no, "spbs"))
-	{
-		failure = "gpio_request";
-		goto error;
-	}
-
-	if(gpio_direction_input(gpio_no))
-	{
-		failure = "gpio_direction_input";
-		goto error_free_gpio;
-	}
+	qca_gpio_init(gpio_no);
```

Comment:
I agree, I also was waiting something drastic to happen, because the original example of use I took from:
```
gpio_node = fdt_path_offset(gd->fdt_blob, "/spi/spi_gpio");
if (gpio_node >= 0) {
	qca_gpio_init(gpio_node);
```
then I have removed getting offset (because I didn't know what for search for) but left init. Seems it's redundant for our case.

Reply:
done